### ISSUE TITLE
Add support provider dify.ai

### ISSUE THREAD

**Issue by Asterovim** [continuedev/continue#1402]
### Validations

- [X] I believe this is a way to improve. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement) that requests the same enhancement

### Problem

The dify.ai is not compatible with continue.dev

Dify.ai is a great provider and an easy tools to construct chatbot : https://github.com/langgenius/dify
https://docs.dify.ai/user-guide/launching-dify-apps/developing-with-apis

### Solution

Add compatibility to https://github.com/langgenius/dify

**Comment by sestinj** [comment]
Looks like a great project. As I probably won't get to this immediately over some pressing issues, do you have any interest in making a PR to add this feature? There is a good tutorial of how to do this in CONTRIBUTING.md

**Comment by guixin567** [comment]
continue + dify = perfect，expect to adapter it

**Comment by Naturalvolume** [comment]
I agree; this is an extremely useful ability. We eagerly anticipate your support for this feature.
